client/src/app/site/pages/organization/pages/accounts/pages/account-list/components/account-merge-dialog/account-merge-dialog.component.html

```diff
@@ -0,0 +1,104 @@
+                                    &middot;&nbsp;{{ user.pronoun }}
+                                </span>
+                            </div>
+                            <div *ngIf="isCrossedOut(user)">
+                                <span class="ellipsis-overflow crossed-out">{{ user.short_name }}</span>
+                                <span class="meta-info user-subtitle" *ngIf="!!user.gender">
+                                    &middot;&nbsp;{{ user.gender }}
+                                </span>
+                                <span class="meta-info user-subtitle" *ngIf="!!user.pronoun">
+                                    &middot;&nbsp;{{ user.pronoun }}
+                                </span>
+                            </div>
+                            <div style="float: right; display: inline-flex">
+                                &nbsp;
+                                <os-icon-container
+                                    icon="account_balance"
+                                    *ngIf="!user.is_physical_person"
+                                    matTooltip="{{ 'Is no natural person' | translate }}"
+                                ></os-icon-container>
+                                <os-icon-container
+                                    icon="block"
+                                    *ngIf="!user.is_active"
+                                    matTooltip="{{ 'Inactive' | translate }}"
+                                ></os-icon-container>
+                                <os-icon-container
+                                    icon="fingerprint"
+                                    *ngIf="user.hasSamlId"
+                                    matTooltip="{{ 'Has SSO identification' | translate }}"
+                                ></os-icon-container>
+                            </div>
+                        </div>
+                        <div class="meta-info user-subtitle ellipsis-overflow">
+                            {{ user.saml_id || user.username }}
+                            <ng-container *ngIf="user.email">&middot; {{ user.email }}&nbsp;</ng-container>
+                            <ng-container *ngIf="user.member_number">&middot; {{ user.member_number }}</ng-container>
+                        </div>
+                        <div class="meta-info user-subtitle ellipsis-overflow" *ngIf="user?.isLastLogin">
+                            {{ 'Last login' | translate }} {{ user.last_login | localizedDate }}
+                        </div>
+                    </div>
+                </td>
+            </ng-container>
+
+            <ng-container matColumnDef="icon">
+                <td mat-cell *matCellDef="let user">
+                    <div class="iconCell">
+                        <os-icon-container
+                            icon="event_available"
+                            [iconTooltip]="userMeetings(user)"
```

Comment:
This might be problematic in terms of performance as the method does processing with arrays. To fix this you could use change detection `OnPush` (preferred) or store and access the meeting names differently.

Reply:
Now use OnPush.